server/routes.js

```diff
@@ -138,7 +138,9 @@ module.exports = (app) => {
   app.get('/groups/tags', groups.getGroupTags); // List all unique tags on all groups
   app.get('/groups/:groupid', aZ.authorizeAccessToGroup({authIfPublic: true}), groups.getOne);
   app.get('/groups/:groupid/users', aZ.authorizeAccessToGroup({authIfPublic: true}), cache(60), groups.getUsers); // Get group users
+  app.get('/groups/:groupid/active_users', aZ.authorizeAccessToGroup({authIfPublic: true}), cache(60), groups.getActiveUsers); // Get group users
```

Comment:
you are right. And we don't need a separate method for this, it should be a filter on the getUsers method in groups controller. I'll rework.